src/data/bool.lean

```diff
@@ -75,6 +75,8 @@ decidable_of_decidable_of_iff or.decidable exists_bool.symm
   cond (to_bool p) t e = if p then t else e :=
 by by_cases p; simp *
 
+theorem coe_bool_iff : ∀ {a b : bool}, (a ↔ b) ↔ a = b := dec_trivial
```

Comment:
Shouldn't this be marked `@[elim_cast]`?  Or would that be a problem because there's `↔` on the left and `=` on the right?

Reply:
The rules on that aren't clear to me. @robertylewis @pnmadelaine ?

Reply:
I think it would be safe. But let's wait until the next round of updates to norm_cast hits.